packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -314,6 +314,18 @@ class MediaQueryData {
 
   /// Whether the device is inverting the colors of the platform.
   ///
+  /// Setting it, for instance by using:
+  ///
+  /// ```dart
```

Comment:
Can you put these in `{@tool snippet}...`
That way the code will be analyzed and if it becomes out of date we'll know to update it. :)